common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -397,6 +397,7 @@ def get_html(self):
             'progress_detail': Progress.to_js_detail_str(progress),
             'content': self.get_problem_html(encapsulate=False),
             'graded': self.graded,
+            'attempts_used': self.attempts,
```

Comment:
Nit: can we rename get_html to something more meaningful (and update the useless comment)? I had to go into the code to understand what HTML was being returned.